src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -811,14 +780,15 @@ public void setMode(TraverseMode mode) {
     public void setModes(TraverseModeSet modes) {
         this.modes = modes;
         if (modes.getBicycle()) {
+//            TODO decide if we want to keep this piece of logic. It might be totally redundant and obsolete.
             // This alternate routing request is used when we get off a bike to take a shortcut and are
             // walking alongside the bike. FIXME why are we only copying certain fields instead of cloning the request?
             bikeWalkingOptions = new RoutingRequest();
             bikeWalkingOptions.setArriveBy(this.arriveBy);
             bikeWalkingOptions.maxWalkDistance = maxWalkDistance;
             bikeWalkingOptions.maxPreTransitTime = maxPreTransitTime;
             bikeWalkingOptions.walkSpeed = walkSpeed * 0.8; // walking bikes is slow
-            bikeWalkingOptions.walkReluctance = walkReluctance * 2.7; // and painful
+//            bikeWalkingOptions.walkReluctance = walkReluctance * 2.7; // and painful
```

Comment:
To zakomentowanie jest potrzebne?

Reply:
Wolę zostawić, bo to fragment z typu "może wybuchnąć w niespodziewanym miejscu".